src/main/features/testing-support/views/create-claim-draft.njk

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import linkButton, csrfProtection, errorSummary %}
+{% from "date.njk" import dateInput %}
+
+{% set title = 'Create Claim Draft | Testing support' %}
+{% set heading = 'Create Claim Draft' %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form method="post" novalidate>
+        {{ csrfProtection(csrf) }}
+
+        <div class="notice">
+          {{ linkButton('Create Claim Draft', 'action[claim]', 'button button-secondary') }}
```

Comment:
A 'action[claim]' is no longer needed since you not using it to read draft type.

Reply:
removed 👍